airflow/jobs/scheduler_job.py

```diff
@@ -951,6 +955,19 @@ def _prepare_simple_dags(self, dags: List[DAG], pickle_dags: bool, session=None)
         return simple_dags
 
 
+def processor_factory(file_path, failure_callback_requests, dag_ids, pickle_dags):
+    """
+    Returns DagFileProcessorProcess instance. Will get pickled when start_method is
+    'spawn', so this can't be an class method.
```

Comment:
Fwiw I think this could be _static_ method, just not an instance method. Check out what we do for SchedulerJob:

https://github.com/apache/airflow/blob/8d6f1aa4b5bb8809ffc55dc0c62e6d0e89f331e5/airflow/jobs/scheduler_job.py#L185-L196

and

https://github.com/apache/airflow/blob/8d6f1aa4b5bb8809ffc55dc0c62e6d0e89f331e5/airflow/jobs/scheduler_job.py#L111-L112

Just thinking it would be good to be consistent.

Reply:
+1